client/src/renderer/redux/actionTypes.js

```diff
@@ -1 +1,2 @@
 /** @format */
+export const ADDCOMMAND = "ADDCOMMAND";
```

Comment:
Please use underscores (`_`) to delimit words in constants.
```suggestion
export const ADD_COMMAND = "ADD_COMMAND";
```